docs/en/ingest-management/index.asciidoc

```diff
@@ -1,13 +1,15 @@
 :doctype: book
 :beats-repo-dir: {beats-root}
-:im-forum: https://discuss.elastic.co/tags/c/elastic-stack/81/stack-ingest-management
+:forum: https://discuss.elastic.co/
 :code-path: {beats-repo-dir}/libbeat/docs/tab-widgets/code
 :elastic-endpoint-integration: Endpoint Security
 
 = Fleet User Guide
 
 include::{docs-root}/shared/versions/stack/{source_branch}.asciidoc[]
 
+:release-state: released
```

Comment:
setting this to released temporarily so you can see the installation instructions.